### PR TITLE
fix: variable name conflicts with built in function

### PR DESCRIPTION
**Pull request by TerryHowe**
**What this PR does / why we need it**:

The variable name `len` conflicts with the built in. This will simplify my other PR a bit as well.

### CODE REVIEW

File: cmd/oras/internal/display/status/progress/manager.go [hunk 1]
```diff
@@ -92,17 +92,17 @@ func (m *manager) render() {
 	defer m.statusLock.RUnlock()
 	// todo: update size in another routine
 	width, height := m.console.Size()
-	len := len(m.status) * 2
+	statusLength := len(m.status) * 2
```
Comment (qweeah) [inline cmd/oras/internal/display/status/progress/manager.go:95]: How about `lineCount`, one status output take two lines.
```suggestion
	lineCount := len(m.status) * 2
```

**Review by qweeah — APPROVED** [review]
LGTM with minor suggestions.

**Comment by qweeah** [comment]
Added a unit test to cover the missing line, but I have wrongly pushed to #1447. Guess this should be merged first and merging #1447 will save that fault.

File: cmd/oras/internal/display/status/progress/manager.go (hunk 1 above)
Comment (TerryHowe) [reply cmd/oras/internal/display/status/progress/manager.go]: Done

**Pull request merged**